framework/db/Command.php

```diff
@@ -90,6 +90,11 @@ class Command extends Component
      */
     private $_pendingParams = [];
     /**
+     * @var mixed INSERT/UPDATE return parameters to be bound to the current PDO statement
+     * or boolean true if return parameters are the query result.
```

Comment:
Comment could be `Either an array of return parameters to be bound to the current PDO statement
or a special value "true" if return parameters are the query result.`